fpdf/template.py

```diff
@@ -84,6 +84,7 @@ def load_elements(self, elements):
             "priority": int,
             "multiline": (bool, type(None)),
             "rotate": (int, float),
+            "keep_aspect_ratio": object,  # "bool or equivalent",
```

Comment:
Why not typing this as `bool`?

Reply:
I was mostly just a little confused here with the types, as other fields also seem to be flags but are typed as objects (eg. `bold `& `italic`). Seems that "multiline" is also possibly a bool but it is typed as `(bool, type(None)`) for some reason.

Reply:
@gmischler just a ping for you on this subject, as you have more experience than me with the templating system 🙂

Reply:
It looks like I added this in #244.
The motivation was to allow other values with a True/False meaning, similar to what most library functions in Python allow. The documentation also specifies "enabled with True or equivalent value".
If there is a better/more explicit way to specify this, I haven't found it yet...

Reply:
Alright!
So what do you think is the best there? `int`? `(bool, int)`?

IMHO we could stick to `bool` even if non-boolean "truthy" values are also valid, this would make the **intent** of this parameter clearer.

Reply:
> IMHO we could stick to `bool` even if non-boolean "truthy" values are also valid, this would make the **intent** of this parameter clearer.

If we use `bool`, then we need an additional conversion (first from `str` to `int`, then to `bool`) for CSV files.

You'd expect there to be a standard way of expressing "bool or True/False equivalent", wouldn't you?
A quick search has not turned up anything useful for me...

Maybe just using `bool` is really the simplest way, although that is technically a backwards incompatible change.